### PR TITLE
[TW-354] Microsoft AD FS SSO

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Updated written instructions and screenshots. Expanded on written instructions where applicable.
* Added a reference to setting up SSO in Postman
* Updated header, making them more descriptive. Also searched the docs for existing references to the previous version of each changed header.
* Wording and grammar changes where applicable
* Updated Markdown syntax for images
* Ran Vale on the page, and found no major issues
* Updated front matter, removing deprecated key-values

### CODE REVIEW

File: src/pages/docs/administration/sso/microsoft-adfs.md [hunk 1]
```diff
@@ -14,187 +11,146 @@ contextual_links:
...
-
-    > You should use a trusted URL encoder to generate the encode values.
-
-1. URL encode each value.
-
-    * **RPID** - `https%3A%2F%2Fidentity-example.getpostman.com`
-    * **Relay State** - `35ef7ab89gh99hh00`
-
-1. Merge the URL encoded values with the string below, and URL encode the whole string.
+1. For Microsoft AD FS 2.0, run IISReset to restart IIS.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/sso/microsoft-adfs.md:145]: Should IISReset be in code font?

**Review by mark-anthony-postman — APPROVED** [review]
Wow, this is a huge improvement over the old page. The new structure and organization are great! 🌟

File: src/pages/docs/administration/sso/microsoft-adfs.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/sso/microsoft-adfs.md]: Yes! I agree. This is now updated.

**Review by asutosh97 — APPROVED** [review]

**Review by claireriley — APPROVED** [review]
LGTM! Thanks @MatthewPinsker-postman ! ⭐

**Pull request merged**